### DISCUSSION TITLE
[FORUM ARCHIVE] Atlas - A Windows 10 Hero Wallpaper Remix

### DISCUSSION THREAD

**Discussion by the-P1neapple** [Atlas-OS/Atlas#1212] [Community Artwork]
# Full Quality is Available Below

Variant 1 (Dimmer Logo, Heavy Smoke)
![1682769817-956751-atlasos-windows10-original-wallpaper-remake.png](https://github.com/user-attachments/assets/95b38d6b-8e73-4fd0-841e-45f9ae88f335)

Variant 2 (More Solid Logo, Heavy Smoke)
![1682768921-898721-atlasos-windows10-original-wallpaper-remake-2.png](https://github.com/user-attachments/assets/8a483706-4895-4c38-879f-de6b8f67b3e2)

Variant 3 (Dimmer Logo, Lighter Smoke)
![1682768923-86766-atlasos-windows10-original-wallpaper-remake-3.png](https://github.com/user-attachments/assets/89761d73-c5bd-42bd-80fe-c5e3c68d1300)

Variant 4 (More Solid Logo, Lighter Smoke)
![1682768924-251820-atlasos-windows10-original-wallpaper-remake-4.png](https://github.com/user-attachments/assets/de8b63e0-05c9-4618-bcaf-a87b5c2c0df1)

Full Quality Downloads (Google Drive):
[Variant 1 (Dimmer Logo, Heavy Smoke)](https://drive.google.com/u/0/uc?id=1N3pS9MjAQBYq3UdsSl7E8dEVEip1FAox&export=download)

[Variant 2 (More Solid Logo, Heavy Smoke)](https://drive.google.com/u/0/uc?id=1PLRRPRzxx84eT0isAic7eoaZM3kScLVO&export=download)

[Variant 3 (Dimmer Logo, Lighter Smoke)](https://drive.google.com/u/0/uc?id=1qxs6NpHNj90f99uWcXisBydDjSU0r_Je&export=download)

[Variant 4 (More Solid Logo, Lighter Smoke)](https://drive.google.com/u/0/uc?id=1iYXhj_QTnQawzqXlCUJyqDQaaZnKPAdL&export=download)

All of this was made by myself in Photoshop

Smoke Overlay: Image by [Freepik](https://www.freepik.com/free-vector/realistic-fog-background_18775337.htm#query=abstract%20smoke&position=11&from_view=keyword&track=ais)

**Comment by Ast3risk-ops** [comment]
I will note that the originals can be uploaded to github (they're under 10 MB) and wouldn't be compressed.
